server/sql.go

```diff
@@ -256,7 +256,9 @@ func (s *Server) registerSQLRoutes(g *echo.Group) {
 			}
 			defer driver.Close(ctx)
 
-			rowSet, err := driver.Query(ctx, exec.Statement, exec.Limit)
+			// Limit SQL query result size.
+			stmt := getStatementWithResultLimit(exec.Statement, exec.Limit)
+			rowSet, err := driver.Query(ctx, stmt, exec.Limit)
```

Comment:
What if we do this in driverutil.go Query()?

Reply:
Done. That's better.